xml/dm_registers.xml

```diff
@@ -37,6 +36,12 @@
 
             7 (other): An access failed for another reason.
         </field>
+	<field name="allhasreset" bits="19" access="R" reset="-">
+          This field is 1 when all currently selected harts have been reset since the last write to its \Fhasresetack.
+        </field>
+        <field name="anyhasreset" bits="18" access="R" reset="-">
```

Comment:
anyhavereset?

Reply:
I guess the problem is that neither "have" nor "has" works with both "any" or "all" grammatically.

"Do you know if any hart has reset in the past 5 minutes?"
"Do you know if all harts have reset in the past 5 minutes?"

`has` is fewer letters...

Reply:
Any harts have reset in the past 5 minutes.
All harts have reset in the past 5 minutes.

But that doesn't work in question form. The other bits solve this by removing the verb altogether, although that creates the question of whether reset is a verb. If we're not going to be able to be 100% clear, maybe just go with "allreset" and "anyreset" to parallel the halted bits. I don't feel strongly about this since I can't find something that really works across the board.